dev-manual/env/compose.rst

```diff
@@ -42,9 +44,14 @@ change your current directory.
 
 .. code-block:: bash
 
-   git clone -b stable/2.4.x https://github.com/artefactual/atom.git atom
+   git clone -b qa/2.5.x https://github.com/artefactual/atom.git atom
    cd atom
 
+.. note::
+
+   Clone the "stable/2.4.x" branch if you want to test the latest stable
```

Comment:
Do we want to specify an exact version here? If so, then we need to remember to review and update this with each major release. Might we suggest cloning the latest stable branch (e.g. `stable/2.4.x`, etc) or something more generic instead?

Reply:
Ideally that could be handled with Sphinx vars, I don't really know a lot about this but I think the AM docs use something like that to specify the latest stable and qa branches. In the time being, that's something that we already have to do one the release cycle when the new 2.6 branch is created. If I remember well it's needed in quite a few places, so one more shouldn't be a problem.

Reply:
Hmm, maybe nevermind, looks like we've been doing this already anyways

Reply:
Oh just saw your reply. Yes, we can look into that in the future. For now I think it's fine.